_shared/project/pyproject.toml

```diff
@@ -59,6 +59,9 @@ omit = [
     {% if cookiecutter.get("_directory") == "pyapp" %}
     "*/{{ cookiecutter.package_name }}/app.py",
     {% endif %}
+    {% if cookiecutter.get("_directory") == "pyramid-app" %}
+    "{{ cookiecutter.package_name }}/pshell.py",
+    {% endif %}
```

Comment:
Add `pshell.py` to the coverage ignore list. I don't think there's much point in unit testing this file: it's only used by `make shell` in dev.